Dockerfile

```diff
@@ -0,0 +1,70 @@
+# BASE IMAGE (37.4MB)
+# ------------------------------------------------------------------------------------
+FROM alpine:3.14 as base
+
+ARG NODEJS_VER=14
+ARG NPM_VER=7
+ARG TINI_VER=0.19
+
+ENV NODEJS_VER=$NODEJS_VER
+ENV NPM_VER=$NPM_VER
+ENV TINI_VER=$TINI_VER
+
+RUN apk add --no-cache nodejs~="$NODEJS_VER" tini~="$TINI_VER"
+RUN apk add --no-cache curl~=7 npm~="$NPM_VER"
```

Comment:
Do we need npm, since you're getting pnpm directly?

Reply:
Removed!